docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -155,7 +161,7 @@ Install Terraform if it is not already installed: visit [terraform.io](https://w
 
 #### Download the Template Files
 
-Follow the instructions you receive when your registration process is complete (see [UID2 Operator Account Setup](#uid2-operator-account-setup)) to download the template files listed in the following table.
+Download the zip file from the [Operator Versions](#operator-versions) above. Be sure to select the GCP Download for the version you are installing. In most circumstances, you should use the latest version. Unzip the files to a convenient location. You will have the files listed in the following table.
```

Comment:
zip > ZIP probably all instances.

Minor mods:

Download the ZIP file listed in [Operator Versions](#operator-versions). Be sure to select the GCP download for the version you are installing. In most circumstances, you should use the latest version. Unzip the files to a convenient location. You will have the files listed in the following table.

But, again, "in most circumstances" we should state the other circumstances too.

Reply:
I changed to:
Download the ZIP file listed in Operator Versions. Be sure to select the GCP download for the latest version. Unzip the files to a convenient location. You will have the files listed in the following table.